cel/README.md

```diff
@@ -0,0 +1,274 @@
+corresponding names of the CEL expressions as provided in the `Parameters`.
+
+```yaml
+Name:         celrun-is-red-8lbwv
+Namespace:    default
+API Version:  tekton.dev/v1alpha1
+Kind:         Run
+Metadata:
+  Creation Timestamp:  2021-01-20T17:51:52Z
+  Generate Name:       celrun-is-red-
+# […]
+Spec:
+  Params:
+    Name:   red
+    Value:  {'blue': '0x000080', 'red': '0xFF0000'}['red']
+    Name:   is-red
+    Value:  {'blue': '0x000080', 'red': '0xFF0000'}['red'] == '0xFF0000'
+  Ref:
+    API Version:         cel.dev/v1alpha1
+    Kind:                CEL
+  Service Account Name:  default
+Status:
+  Completion Time:  2021-01-20T17:51:52Z
+  Conditions:
+    Last Transition Time:  2021-01-20T17:51:52Z
+    Message:               CEL expressions were evaluated successfully
+    Reason:                EvaluationSuccess
+    Status:                True
+    Type:                  Succeeded
+  Extra Fields:            <nil>
+  Observed Generation:     1
+  Results:
+    Name:      red
+    Value:     0xFF0000
+    Name:      is-red
+    Value:     true
+  Start Time:  2021-01-20T17:51:52Z
+Events:
+  Type    Reason         Age   From            Message
+  ----    ------         ----  ----            -------
+  Normal  RunReconciled  13s   cel-controller  Run reconciled: "default/celrun-is-red-8lbwv"
+```
+
+If no CEL expressions are provided, any CEL expression is invalid or there's any other error, the `CEL` `Custom Task`
+will fail and the details will be included in `status.conditions` as such:
+
+```yaml
+Name:         celrun-is-red-4ttr8
+Namespace:    default
+API Version:  tekton.dev/v1alpha1
+Kind:         Run
+Metadata:
+  Creation Timestamp:  2021-01-20T17:58:53Z
+  Generate Name:       celrun-is-red-
+# […]
+Spec:
+  Ref:
+    API Version:         cel.dev/v1alpha1
+    Kind:                CEL
+  Service Account Name:  default
+Status:
+  Completion Time:  2021-01-20T17:58:53Z
+  Conditions:
+    Last Transition Time:  2021-01-20T17:58:53Z
+    Message:               Run can't be run because it has an invalid spec - missing field(s) params
+    Reason:                RunValidationFailed
+    Status:                False
+    Type:                  Succeeded
+  Extra Fields:            <nil>
+  Observed Generation:     1
+  Start Time:              2021-01-20T17:58:53Z
+Events:                    <none>
+```
+
+For more information about monitoring `Run` in general, read [monitoring execution status](https://github.com/tektoncd/pipeline/blob/master/docs/runs.md#monitoring-execution-status).
+
+### Using the evaluation results
+
+A successful `Run` contains the `Results` of evaluating the CEL expressions under `status.results`, with the name of
+each evaluation `Result` matching the name of the corresponding CEL expression as provided in the `Parameters`.
+Users can reference the `Results` in subsequent `Tasks` using variable substitution, as such:
```

Comment:
hey @jerop sorry for the late comment, just wondering' what's the type and structure of the result? (e.g. json? or some CEL specific thing? maybe we can link to some CEL docs about this?)